node/src/eth.rs

```diff
@@ -172,10 +173,10 @@ pub async fn spawn_frontier_tasks<RuntimeApi, Executor>(
                 fc_mapping_sync::sql::SyncWorker::run(
                     client.clone(),
                     backend,
-                    Arc::new(b),
+                    b.clone(),
                     client.import_notification_stream(),
                     fc_mapping_sync::sql::SyncWorkerConfig {
-                        read_notification_timeout: Duration::from_secs(10),
+                        read_notification_timeout: Duration::from_secs(30), // TODO: tbd
```

Comment:
Why’s this changed?

Reply:
Just synced with value from template. Looks like we can keep our old 10 secs.

Reply:
Yeah, let’s keep the old value